README.md

```diff
@@ -1,4 +1,4 @@
 mapbox-studio-default-fonts
 ===========================
 
-Copyright information for each font is included in each folder in a file named `LICENSE.txt`.
+Copyright information for each font is included in each folder in a file named `LICENSE.txt`/`OFL.txt`/`UFL.txt`/`ofl.markdown`/`Open Font License.markdown`.
```

Comment:
The licenses have different names and formats, might make sense to rephrase that to be more generic or alternatively rename all license files to `LICENSE.xxx`